tests/connection/test_connection_service.py

```diff
@@ -570,11 +540,7 @@ def test_build_connection_response(self):
         server_name = 'testserver'
         db_name = 'testdb'
         user = 'testuser'
-        mock_connection = MockConnection({
-            'host': server_name,
-            'dbname': db_name,
-            'user': user
-        })
+        mock_connection = MockServerConnection(cur = None, host = server_name, name = db_name, user = user)
```

Comment:
You should pass a version string here in mockServerConnection and below in assert just use that string, you won't have to convert tuple to string that way and we will also test that the version that we send is returned in response instea dof the default.